cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
-    path("__debug__/", include("debug_toolbar.urls")),
+    path("api/user/", include("user.urls", namespace="user")),
+    # path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The Debug Toolbar URL pattern is commented out. Ensure that this is conditionally included only in development environments to avoid exposing it in production. You can use an environment check to include this URL pattern when `DEBUG` is `True`.